bundles/org.openhab.binding.energidataservice/src/main/java/org/openhab/binding/energidataservice/internal/action/EnergiDataServiceActions.java

```diff
@@ -63,7 +65,7 @@ public class EnergiDataServiceActions implements ThingActions {
 
     private @Nullable EnergiDataServiceHandler handler;
 
-    @RuleAction(label = "@text/action.get-prices.label", description = "@text/action.get-prices.description")
+    @RuleAction(label = "@text/action.get-prices.label", description = "@text/action.get-prices.description", id = "getPrices1", visibility = Visibility.HIDDEN)
     public @ActionOutput(name = "prices", type = "java.util.Map<java.time.Instant, java.math.BigDecimal>") Map<Instant, BigDecimal> getPrices() {
```

Comment:
This should fix the warning „ Method EnergiDataServiceActions::getPrices has a single output but does not use the default output name in the ActionOutput annotation. This should be fixed in the binding.“:

```suggestion
    public @ActionOutput(type = "java.util.Map<java.time.Instant, java.math.BigDecimal>") Map<Instant, BigDecimal> getPrices() {
```

Reply:
You're right, I will test that.

Reply:
Done